### PR TITLE
docs: add how to translate info boxes headers

### LINKED ISSUE #46852: docs: add to the language lead handbook how to translate tip,warning,attention,note box headers
It can be translated from the file https://github.com/freeCodeCamp/freeCodeCamp/blob/main/docs/index.html

The process should be added to the language lead handbook

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46852 

This add how to translate the headers of these boxes in the translated docs:

image

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]

File: docs/language-lead-handbook.md [hunk 1]
```diff
@@ -94,6 +94,86 @@ You can convert from one format to the other carefully changing it manually. Or
...
+  },
+  attention: {
+    label: {
+      '/': 'Attention'
+    }
+  }
+}
+```
+
+Inside the object of the label property, before the `'/'` property, you would add a new property for your language, like `/i18n/<langauge>/`.
```
Comment (Sembauke) [inline docs/language-lead-handbook.md:144]:
```suggestion
Inside the object of the label property, before the `'/'` property, you would add a new property for your language, like `/i18n/<language>/`.
```

**Review by Sembauke — DISMISSED** [review]

File: docs/language-lead-handbook.md (hunk 1 above)
Comment (ieahleen) [reply docs/language-lead-handbook.md:144]: lol, thanks

**Review by DanielRosa74 — APPROVED** [review]
LGTM

**Review by Sboonny — APPROVED** [review]

**Comment by Sboonny** [comment]
LGTM, thank you.

**Pull request merged**